### PR TITLE
Add renaming of whitelisted_jwt.rb model file

### PR DESCRIPTION
**Pull request by markhallen**
Failure to rename the file results in the error `uninitialized constant User::AllowlistedJwt`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ For `Allowlist`, you need to update the `include` line you're using in your user
 include Devise::JWT::RevocationStrategies::Allowlist
 ```
 
-You also have to rename your `WhitelistedJwt` model to `AllowlistedJwt` and change the underlying database table to `allowlisted_jwts` (or configure the model to keep using the old name).
+You also have to rename your `WhitelistedJwt` model to `AllowlistedJwt`, rename `model/whitelisted_jwt.rb` to `model/whitelisted_jwt.rb` and change the underlying database table to `allowlisted_jwts` (or configure the model to keep using the old name).
```
Comment (waiting-for-dev) [inline README.md:44]: It should be `model/allowlisted_jwt.rb` :slightly_smiling_face:

**Review by waiting-for-dev — COMMENTED** [review]
Hey, thanks for your contribution. However, there's a typo in the new name of the file. Could you fix it, please?

File: README.md (hunk 1 above)
Comment (markhallen) [reply README.md]: Ooops. Fixed. Thanks

**Comment by waiting-for-dev** [comment]
Thanks!!

**Pull request merged**